### PR TITLE
PT-2072: Collapse/expand arrows in the age of onset category do not work

### PR DESCRIPTION
**Pull request by mjshepherd**

### CODE REVIEW

**Comment by mjshepherd** [comment]
@veronikaslc @antonkatz Please review

**Comment by veronikaslc** [comment]
The symbol that you are using is ▶, 0x25b6 converted to Hex, it just looks different/weird in my browser. Across the phenotips we are using ► which is  0x25ba.

File: components/vocabularies/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml [hunk 1]
```diff
@@ -315,9 +362,17 @@ $xwiki.jsx.use('PhenoTips.HPOBranchDisplayer')##
 }
 .hpo-branch .term-entry .info-tool {
   cursor: help;
+}
+.hpo-branch .term-entry .expand-tool {
+    margin-left: -1em;
+    position: absolute;
+    top: .5em;
```
Comment (veronikaslc) [inline components/vocabularies/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml]: It looks too low in my browser, we may be do need that line?

File: components/vocabularies/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml [hunk 2]
```diff
@@ -200,6 +200,49 @@ $xwiki.jsx.use('PhenoTips.HPOBranchDisplayer')##
     });
     $$('.hpo-branch .term-entry:not(.initialized)').invoke('addClassName', 'initialized');
   });
+});
+document.observe('xwiki:dom:loaded', function(){
+  function canCollapse(sublist){
+    return sublist.select('input').every(function(input){
+      return !input.checked
```
Comment (veronikaslc) [inline components/vocabularies/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml]: can I be too annoying and say its missing a semicolon? :)

**Comment by veronikaslc** [comment]
If you click on the info icon of any of expanded items, the expanded part suddenly collapses.

File: components/vocabularies/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml [hunk 3]
```diff
@@ -200,6 +200,49 @@ $xwiki.jsx.use('PhenoTips.HPOBranchDisplayer')##
...
+        e.stop();
+        if (sublist.hasClassName('collapsed')) {
+          sublist.removeClassName('collapsed');
+          expandTool.innerHTML = "▼";
+        } else if (canCollapse(sublist)) {
+          expandTool.innerHTML = "▶";
+          sublist.addClassName('collapsed');
+        }
+      });
+      var toggleCollapse = function() {
```
Comment (antonkatz) [inline components/vocabularies/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml]: Maybe this function should be outside the forEach loops?

**Comment by antonkatz** [comment]
+1 Except for that bug with the list collapsing upon clicking the info icon

**Comment by mjshepherd** [comment]
@veronikaslc Let me know how it looks now. I changed the character and the positioning css

File: components/vocabularies/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml (hunk 3 above)
Comment (mjshepherd) [inline components/vocabularies/hpo/ui/src/main/resources/PhenoTips/HPOBranchDisplayer.xml]: Right now its relying on a bunch of variables within the loop. I could do some closure thing but imho this is more readable.

**Comment by marta-** [comment]
@veronikaslc @allasm please re-reveiw.

**Comment by veronikaslc** [comment]
+1.

**Pull request merged**